cookiecutter/config.py

```diff
@@ -23,10 +23,16 @@
 
 USER_CONFIG_PATH = os.path.expanduser('~/.cookiecutterrc')
 
+BUILTIN_ABBREVIATIONS = {
+    'gh': 'https://github.com/{0}.git',
+    'bb': 'https://bitbucket.org/{0}',
+}
+
```

Comment:
Unrelated to this pull request, but I would love to see GitLab added this list. I'll open a new issue.